src/baseApi.ts

```diff
@@ -1,7 +1,11 @@
 import got, { HTTPError, Method } from 'got'
 import * as jwt from 'jsonwebtoken'
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const { version: libVersion } = require('../package.json')
+
+const {
+  version: pluggyNodeVersion,
+  dependencies: { got: gotVersion },
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+} = require('../package.json')
```

Comment:
Yep, the path continues working.... (remember that we used to do it like this, I also tested it). 
I use require since I had problems importing it with `import`